cli/src/utils/buildPSP.ts

```diff
@@ -147,17 +147,15 @@ export async function buildPSP(
   programName: string
 ) {
   let circuitFileName = findLightFile(circuitDir);
-  // let programName = "verifier"
   console.log("Creating circom files");
   const macroCircomBinPath = path.resolve(__dirname, "../../bin/macro-circom");
-  // TODO: check whether macro circom binary exists if not fetch it
   // TODO: check whether circom binary exists if not load it
   const dirPath = path.resolve(__dirname, "../../bin/");
 
   await downloadFileIfNotExists(
```

Comment:
4 params; would be nice to put these into an object else easy to mess up passing 
e.g.
```
downloadFileIfNotExists({
  macroCircomBinPath,
  dirPath,
  repoName: 'macro-circom',
  fileName: 'macro-circom'
})
```

Reply:
Good idea! Done